plugins/module_utils/network/ios/rm_templates/vrf_address_family.py

```diff
@@ -0,0 +1,2226 @@
+                    },
+                },
+            },
+        },
+        {
+            "name": (
+                "route_replicate.from.vrf.vrf_name.unicast.bgp.asn.route_map"
+            ),
+            "getval": re.compile(
+                r"""
+                ^vrf\sdefinition\s(?P<name>\S+)
+                \saddress-family
+                \s(?P<afi>\S+)
+                (\s(?P<safi>\S+))?
+                \s+route-replicate\sfrom\svrf\s(?P<vrf_name>\S+)\sunicast\sbgp\s(?P<asn>\d+)\sroute-map\s(?P<route_map>\S+)
+                $""", re.VERBOSE,
+            ),
+            "setval": (
+                "route-replicate from vrf {{ route_replicate.from.vrf.vrf_name }} "
+                "unicast all route-map {{ route_replicate.from.vrf.vrf_name.unicast.all.route_map }} "
+                "unicast bgp {{ route_replicate.from.vrf.vrf_name.unicast.bgp.asn }} "
+                "route-map {{ route_replicate.from.vrf.vrf_name.unicast.bgp.asn.route_map }}"
+            ),
+            "result": {
+                '{{ name }}': {
+                    'name': '{{ name }}',
+                    "address_families": {
+                        '{{ "address_families_" + afi + '
+                        '("_" + safi if safi is defined else "_unicast") }}': {
+                            "afi": "{{ afi }}",
+                            "safi": (
+                                "{{ safi if safi is defined else "
+                                "'unicast' }}"
+                            ),
+                            "route_replicate": {
+                                "from_config": {
+                                    "vrf": {
+                                        "name": "{{ vrf_name }}",
+                                        "unicast": {
+                                            "bgp": {
+                                                "as_number": "{{ asn }}",
+                                                "route_map": "{{ route_map }}",
+                                            },
+                                        },
+                                    },
+                                },
+                            },
+                        },
+                    },
+                },
+            },
+        },
+        {
+            "name": (
+                "route_replicate.from.vrf.vrf_name.unicast.connected.route_map"
+            ),
+            "getval": re.compile(
+                r"""
+                ^vrf\sdefinition\s(?P<name>\S+)
+                \saddress-family
+                \s(?P<afi>\S+)
+                (\s(?P<safi>\S+))?
+                \s+route-replicate\sfrom\svrf\s(?P<vrf_name>\S+)\sunicast\sconnected\sroute-map\s(?P<route_map>\S+)
+                $""", re.VERBOSE,
+            ),
+            "setval": "route-replicate from vrf {{ route_replicate.from.vrf.vrf_name }} unicast connected route-map {{ route_replicate.from.vrf.vrf_name.unicast.connected.route_map }}",
```

Comment:
to keep the sanity green , you can try
"setval": "route-replicate from vrf {{ route_replicate.from.vrf.vrf_name }}" \
          " unicast connected route-map {{ route_replicate.from.vrf.vrf_name.unicast.connected.route_map }}",
         
          OR 
          
          "setval": (
        "route-replicate from vrf {{ route_replicate.from.vrf.vrf_name }} "
        "unicast connected route-map "
        "{{ route_replicate.from.vrf.vrf_name.unicast.connected.route_map }}"
    ),